src/app-layout/__tests__/trigger-button.test.tsx

```diff
@@ -92,12 +94,33 @@ describe('Visual refresh trigger-button (not in appLayoutWidget toolbar)', () =>
 
   describe.each([true, false])('Toolbar trigger-button with isMobile=%s', isMobile => {
     describe.each([true, false])('AppLayoutInternals with hasOpenDrawer=%s', hasOpenDrawer => {
+      testIf(!isMobile)('applies the correct class when selected', () => {
+        const ref: React.MutableRefObject<ButtonProps.Ref | null> = React.createRef();
```

Comment:
Is this ref really needed here? You pass it down but never read

Reply:
Will explore in a followup. Thanks for pointing out.